app/services/list_tasks.rb

```diff
@@ -0,0 +1,21 @@
+class ListTasks < ListCollection
+
+  attr_defaultable :task_respository, -> { Task }
+  attr_defaultable :result_serializer, -> { V1::TaskSerializer }
+
+  def initialize page: 1, page_size: 25, project_id: nil
+    @page = page
+    @page_size = page_size
+    @project_id = project_id
+    @errors = []
```

Comment:
Since you are subclassing `ListCollection` it probably would make sense to do less work in this initializer and use `super` to get the desired behavior from the super class.